src/Error/DefaultErrorHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tomaj\NetteApi\Error;
+
+use Nette\Http\Response;
+use Throwable;
+use Tomaj\NetteApi\Output\Configurator\ConfiguratorInterface;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tracy\Debugger;
+
+final class DefaultErrorHandler implements ErrorHandlerInterface
+{
+    /** @var ConfiguratorInterface @inject */
+    public $outputConfigurator;
```

Comment:
Toto ti funguje? Nemalo by, inject ide len v presenteoch, tu musíš ísť cez konstruktor

Reply:
Funguje. Ale prerobim na constructor pre istotu.

Reply:
zmenis na private? ci potrebujes public? public to bolo kvoli tomu injectu predpokladam, teraz asi nie je dovod